packages/dd-trace/src/config.js

```diff
@@ -175,6 +188,37 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_TRACEPARENT_ENABLED,
       false
     )
+    // TODO: change this to traceparent,datadog
+    // Traceparent should be prioritized over datadog headers, but this is a breaking change until
+    // we support tracestate headers too, otherwise we would lose origin and tags header data.
+    const defaultPropagationStyle = ['datadog']
+    if (isTrue(DD_TRACE_TRACEPARENT_ENABLED)) {
+      defaultPropagationStyle.push('tracecontext')
+    }
+    if (isTrue(DD_TRACE_B3_ENABLED)) {
+      defaultPropagationStyle.push('b3')
+      defaultPropagationStyle.push('b3 single header')
```

Comment:
I saw a discussion about changing this. Are we keeping this value?

Reply:
Yes, but we will also have b3multi. In a future major, the meaning of `b3` will change to mean single-header form to match with how OTel defines it.